apps/_pomodoro_app/realtime/pomodoroExpressBackend.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+
+const {generateLink} = require('./application/utils')
```

Comment:
```suggestion
const { generateLink } = require('./application/utils')
```

Reply:
Made utils/generateLink a separate file, and module.exports = generateLink